profiles/views.py

```diff
@@ -220,3 +221,41 @@ def form_valid(self, form):
 
 def about(request, template_name='about.html'):
     return render(request, template_name,)
+
+
+class BulkUser(LoginRequiredMixin, View):
+    '''
+    Custom class to create bulk student users.
+    '''
+
+    def get(self, request):
+        username = request.user
+
+        if username.is_superuser:
+            return render(request, "bulkuserform.html")
+        else:
+            return render(request, "404.html")
+
+    def post(self, request):
+        username = request.user
+
+        if username.is_superuser:
+            format = request.POST.get("format")
+            user_start = int(request.POST.get("start"))
+            user_end = int(request.POST.get("end"))
+            branch = request.POST.get("branch")
+            year = request.POST.get("year")
+            course = request.POST.get("course")
+            group = Group.objects.get(name="student")
+
+            if user_start < user_end:
+                user_no = ["%.3d" % users for users in range(user_start, user_end+1)]
+                for student in user_no:
+                    new_user = User.objects.create_user(username=format+str(student), password=str(format)+str(student))
```

Comment:
@RishabhJain2018 can you tell me an example of what would be the values of `format` and `student` vairable and hence what would be the username that would be generated.  I can see that `username=format+str(student)` and I am assuming that `format=12cse` and `student=020` for some example. Am I getting this correctly ?